integration_tests/e2e/cellMoveTemporaryMove.cy.ts

```diff
@@ -101,7 +101,7 @@ context('Move someone temporarily out of a cell', () => {
         .first()
         .should('have.text', 'View cell history for John Smith')
         .should('have.attr', 'href')
-        .should('include', '/prisoner/A1234BC/cell-history')
+        .should('include', 'http://localhost:3101/prisoner/A1234BC/location-details')
```

Comment:
hmmm, I'm feeling the host shouldn't be part of the assertion

Reply:
we need to know that it's pointing to the external prisoner API and not just internally to `/prisoner...` like ti was before